rq/worker.py

```diff
@@ -477,7 +477,11 @@ def work(self, burst=False, logging_level="INFO", date_format=DEFAULT_LOGGING_DA
                     self.execute_job(job, queue)
                     self.heartbeat()
 
-                    did_perform_work = True
+                    completed_jobs += 1
+                    if max_jobs is not None:
+                        if completed_jobs >= max_jobs:
+                            self.log.info("RQ worker %r done, quitting", self.key)
```

Comment:
Could you please change the message to `Worker %s: finished executing <MAX_JOBS> jobs, quitting...` to make it more consistent with the other messages?

Reply:
Yup, I can make that change. However it looks like there is a bit of inconsistency already. 

These 2 messages use `RQ Worker` with the prefix before the name.
https://github.com/rq/rq/blob/2baef02dbd9d1522cd9653f0086e4162d502ef29/rq/worker.py#L451
https://github.com/rq/rq/blob/2baef02dbd9d1522cd9653f0086e4162d502ef29/rq/worker.py#L473

Then there is this message for the exceptional case.
https://github.com/rq/rq/blob/2baef02dbd9d1522cd9653f0086e4162d502ef29/rq/worker.py#L490-L493

What do you think about using:
```suggestion
                            self.log.info("RQ Worker %r finished executing %d jobs, quitting", self.key, completed_jobs)
```

And changing the exception message to:
```python
self.log.error(
    'RQ Worker %r: found an unhandled exception, quitting...',
    self.key, exc_info=True
)
```

Reply:
Ive made that change in the meantime.